src/core/tritonserver.cc

```diff
@@ -1548,6 +1676,22 @@ TRITONSERVER_InferenceRequestSetRequestedOutputClassificationCount(
   return nullptr;  // Success
 }
 
+TRITONSERVER_Error*
+TRITONSERVER_InferenceRequestRequestedOutputClassificationCount(
```

Comment:
Why do we need these special classification methods. Triton should just produce the output tensor as a string tensor and then the endpoint code can access it like any other tensor.

Reply:
So the TRITONSERVER_OutputShape / Data / Datatype APIs should be used for both raw and class outputs? Then the checking on whether the output is requested as class results will be moved inside those TRITONSERVER APIs?

Reply:
Yes. Someone on the "outside" of the TRITONSERVER APIs should just use the same APIs to read the output tensors... but if they requested class results then they will get an output that is the one-dim string tensor instead of what the model configuration specifies as the actual output type/shape.